src/StoreApi/Routes/Checkout.php

```diff
@@ -574,4 +603,38 @@ protected function get_request_payment_data( \WP_REST_Request $request ) {
 
 		return $payment_data;
 	}
+
+	/**
+	 * Order processing relating to customer account.
+	 *
+	 * - Creates a customer account as needed (based on request & store settings).
+	 *   - If so, updates the order with the new customer ID.
+	 * - Updates the order with user details (e.g. address).
+	 *
+	 * @param \WC_Order        $order   Order object.
+	 * @param \WP_REST_Request $request Request object.
+	 */
+	protected function process_customer( \WC_Order $order, \WP_REST_Request $request ) {
+		$order_controller = new OrderController();
```

Comment:
I know this pattern wasn't established in this pr. But as a follow-up issue, I'd like to see us inject `OrderController` and `CartController` as dependencies for this class (I would make it a requirement before making the StoreAPI public).